cmd/machine-controller-manager/app/controllermanager.go

```diff
@@ -208,7 +208,7 @@ func StartControllers(s *options.MCMServer,
 	recorder record.EventRecorder,
 	stop <-chan struct{}) error {
 
-	klog.V(5).Info("Getting available resources")
+	klog.V(4).Info("Getting available resources")
```

Comment:
log 5 , was printing `lease` acquiring logs from the library we use for leader election, which was filling up the logs.. thats why I shifted these logs to V(4), as it will help in easier debugging later.